{
  "uid" : "9c06a3017498e413",
  "name" : "reverts on transfer",
  "fullName" : "Contract: SafeERC20 with address that has no contract code.reverts on transfer",
  "historyId" : "60ed82f00eb27bef214c0d4e29548221",
  "time" : {
    "start" : 1635782730942,
    "stop" : 1635782731905,
    "duration" : 963
  },
  "description" : "",
  "descriptionHtml" : "",
  "status" : "broken",
  "statusMessage" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a04180a0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
  "statusTrace" : "Error: Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a04180a0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))\n                    at Context.<anonymous> (test/token/ERC20/utils/SafeERC20.test.js:13:48)\n                    at processImmediate (node:internal/timers:464:21)",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ ],
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "suite",
    "value" : "Contract: SafeERC20 with address that has no contract code"
  }, {
    "name" : "testClass",
    "value" : "Contract: SafeERC20 with address that has no contract code"
  }, {
    "name" : "testMethod",
    "value" : "reverts on transfer"
  }, {
    "name" : "package",
    "value" : "Contract: SafeERC20 with address that has no contract code"
  }, {
    "name" : "resultFormat",
    "value" : "allure1"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ {
      "uid" : "c4c2dc15cf9a406e",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0680400>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782609049,
        "stop" : 1635782609610,
        "duration" : 561
      }
    }, {
      "uid" : "14f92afbc607b95f",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a07c37f0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782607851,
        "stop" : 1635782608412,
        "duration" : 561
      }
    }, {
      "uid" : "9b6d29fcc14f55b1",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a17de520>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782607405,
        "stop" : 1635782608000,
        "duration" : 595
      }
    }, {
      "uid" : "f35f4a3420e235f0",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a1840f70>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782584704,
        "stop" : 1635782585667,
        "duration" : 963
      }
    }, {
      "uid" : "21070ffeff20743e",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0427610>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782580825,
        "stop" : 1635782581381,
        "duration" : 556
      }
    }, {
      "uid" : "56f17457e015bfc0",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a063e0d0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782528502,
        "stop" : 1635782529066,
        "duration" : 564
      }
    }, {
      "uid" : "cbbb660342a3e78d",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a042d340>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782516603,
        "stop" : 1635782517165,
        "duration" : 562
      }
    }, {
      "uid" : "bf6fb8c51458098a",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0637100>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782514132,
        "stop" : 1635782514689,
        "duration" : 557
      }
    } ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 4,
        "skipped" : 0,
        "passed" : 0,
        "unknown" : 0,
        "total" : 4
      },
      "items" : [ {
        "uid" : "8f8d000749af474c",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/8//#testresult/8f8d000749af474c",
        "status" : "broken",
        "statusDetails" : "Returned error: Transaction simulation failed: Error processing Instruction 0: invalid account data for instruction",
        "time" : {
          "start" : 1635590316938,
          "stop" : 1635590327256,
          "duration" : 10318
        }
      }, {
        "uid" : "3b940ffd749c2ad7",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/7//#testresult/3b940ffd749c2ad7",
        "status" : "broken",
        "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f238be7bbe0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
        "time" : {
          "start" : 1635555005417,
          "stop" : 1635555006419,
          "duration" : 1002
        }
      }, {
        "uid" : "91dabdd32ca9b569",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/6//#testresult/91dabdd32ca9b569",
        "status" : "broken",
        "statusDetails" : "HH110: Invalid JSON-RPC response received: <html>\n                <head><title>502 Bad Gateway</title></head>\n                <body>\n                <center><h1>502 Bad Gateway</h1></center>\n                <hr><center>nginx/1.18.0 (Ubuntu)</center>\n                </body>\n                </html>\n                ",
        "time" : {
          "start" : 1635551255619,
          "stop" : 1635551255683,
          "duration" : 64
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "9c06a3017498e413.json",
  "parameterValues" : [ ]
}